### PR TITLE
Fix #1295: The file upload limit has been increased to 10 megabytes

### LINKED ISSUE #1295: Increase the data upload limit.
## Current Behaviour
By default django supports only 2.5 MB of the data to be uploaded on the web app. Refer [here](https://docs.djangoproject.com/en/1.10/ref/settings/#data-upload-max-memory-size) for more info.

## Effects
Due to the low upload limit the file in challenge creation using zip isn't being uploaded on the app as the size exceeds.

## Expected Behaviour
The upload limit must be increased to 10MB.

### PR DESCRIPTION
**Pull request by memclutter**
Fix #1295.

### CODE REVIEW

File: settings/common.py [hunk 1]
```diff
@@ -273,3 +273,8 @@
     },
     'SUBMISSION_QUEUE': 'submission_task_queue',
 }
+
+# The maximum size in bytes for request body
+# https://docs.djangoproject.com/en/1.10/ref/settings/#data-upload-max-memory-size
+FILE_UPLOAD_MAX_MEMORY_SIZE = 10485760
+DATA_UPLOAD_MAX_MEMORY_SIZE = 10485760
```
Comment (RishabhJain2018) [inline settings/common.py]: Can you please write in comment the size in `MB` for future reference like `10485760 # 10MB`.

Comment (memclutter) [reply settings/common.py]: ok, one moment

File: settings/common.py [hunk 2]
```diff
@@ -273,3 +273,8 @@
     },
     'SUBMISSION_QUEUE': 'submission_task_queue',
 }
+
+# The maximum size in bytes for request body
+# https://docs.djangoproject.com/en/1.10/ref/settings/#data-upload-max-memory-size
+FILE_UPLOAD_MAX_MEMORY_SIZE = 10485760  # 10 MB
+DATA_UPLOAD_MAX_MEMORY_SIZE = 10485760  # 10 MB
```
Comment (RishabhJain2018) [inline settings/common.py]: @memclutter Just a small change, please change the limit to `500 MB`.

**Comment by RishabhJain2018** [comment]
@memclutter Any update on this? We need to merge it asap as it hinders the `challenge creation feature`.

File: settings/common.py (hunk 2 above)
Comment (taranjeet) [reply settings/common.py]: @RishabhJain2018 : 500 mb is too much? Also you will need to increase the limit at Nginx as well

@deshraj

Comment (RishabhJain2018) [reply settings/common.py]: @taranjeet I know, but challenge file can be of `500 MB` as @deshraj suggested. We can increase the limit at nginx too.

**Comment by deshraj** [comment]
Closing this one due to inactivity.

**Pull request closed without merging**